backend/main/src/Services/InvitationHashService/InvitationHashService.ts

```diff
@@ -127,7 +118,49 @@ export class InvitationHashService implements IGenerateAnInvitationHashService {
 			expiryDatetime,
 		});
 	}
+
+	/**
+	 * This method join a user to the group using an invitation hash.
+	 * This method checks the existence of the invitation hash and the user.
+	 * If the invitation hash and the user exists, this method adds the user to the group.
+	 * And return the groupId of the group.
```

Comment:
The comment is also going to be updated to mention that this function checks If the link's expiry datetime is over.